### PR TITLE
geany-status.m4:replace hyphens with text

### LINKED ISSUE #2054: on OpenBSD, Use of "seq" causes "not found" message when configure shows status
https://github.com/geany/geany/blob/c825b581bfabf6912eeb737a4af607e70c6c07a9/m4/geany-status.m4#L30

Due to `ksh` being the default shell on OpenBSD. But changing that line to something like

    for i in {1..$total};

may be a a good solution, if that's shell-portable.

```
config.status: executing libtool commands
config.status: executing default-1 commands
config.status: executing po/stamp-it commands
./configure[22990]: seq: not found

Install Geany in                 : /usr/local
Compiling Git revision           : cd267de2
Using GTK version                : 2.24.32
Enable binary relocation         : no
Build with plugin support        : yes
Use (UNIX domain) socket support : yes
Use virtual terminal support     : yes
Build HTML documentation         : yes
Build PDF documentation          : no
Build API documentation          : no
Generate GtkDoc header           : no

Configuration is done OK.
```

### PR DESCRIPTION
**Pull request by andy5995**
closes #2054

### CODE REVIEW

**Comment by elextr** [comment]
code LGBI, but I would have said why you are removing the dashes "because seq isn't available on some BSDs" or similar, but that can go in the merge message

File: m4/geany-status.m4 [hunk 1]
```diff
@@ -27,7 +27,8 @@ EOF
 	# Print a nice top bar
 	# description + ' : ' + value
 	total=`expr $dlen + 3 + $vlen`
-	for i in `seq 1 $total`; do printf '-'; done
+	echo
+	echo "    Build configuration result:"
```
Comment (b4n) [inline m4/geany-status.m4]: Sadly `printf '%-*s' $total '' | tr ' ' -` isn't portable…

But what about this?
```shell
i=0; while test $i -lt $total; do printf '-'; i=`expr $i + 1`; done
```

Comment (elextr) [reply m4/geany-status.m4]: what about echo "----------------------------------------"

thats guaranteed to be portable (and what configure.in had a long time ago :)

Comment (andy5995) [reply m4/geany-status.m4]: > what about echo "----------------------------------------"
> 
> thats guaranteed to be portable (and what configure.in had a long time ago :)

That seems like a good choice, imo. Let me know when there's agreement and I'll make the change.

**Comment by andy5995** [comment]
Pretty sure that #2060 from @b4n will be used instead. closing...

**Pull request closed without merging**